runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -199,10 +212,13 @@ public void gc(long trimMark) {
         // meets the conditions for direct access.
         long ts = lock.tryOptimisticRead();
         if (ts != 0) {
-            try {
-            if (directAccessCheckFunction.apply(this)) {
-                log.trace("Access [{}] Direct (optimistic-read) access at {}",
-                        this, getVersionUnsafe());
+            Timer optimisticReadTimer = VloMetricsParams
+                    .metrics.timer(VloMetricsParams.VLO_OPTIMISTIC_READ);
```

Comment:
Can you please provide `VloMetricsParams.getOptimisticReadTimer()` instead of this

Reply:
ok